deploy/vagrant/conf/vb.yml

```diff
@@ -0,0 +1,2 @@
+# Memory for each virtual machine
+MachineMemory: 1024 #MB
```

Comment:
Should probably make this > 2GB since that is the default Tachyon worker memory? Ideally the user should be able to run the command without any modification to the default and have it be successful.